packages/loader/container-utils/src/test/error.spec.ts

```diff
@@ -9,7 +9,7 @@ import { strict as assert } from "assert";
 import { ContainerErrorType } from "@fluidframework/container-definitions";
 import { isILoggingError, LoggingError } from "@fluidframework/telemetry-utils";
 import { ISequencedDocumentMessage } from "@fluidframework/protocol-definitions";
-import { CreateContainerError, CreateProcessingError, DataProcessingError, GenericError } from "../error";
+import { CreateContainerError, CreateProcessingError, GenericError } from "../error";
 
 describe("Errors", () => {
     describe("GenericError coercion via CreateContainerError", () => {
```

Comment:
These test changes show the functional difference between `CreateContainerError` and `normalizeError`.

They are broken in the latest iteration, but useful for discussion :)